packages/engine/tests/simulations/sugarscape/README.md

```diff
@@ -0,0 +1,23 @@
+# Antiquities Act
```

Comment:
This doesn't seem like the Sugarscape README?

Reply:
Indeed it's from https://hash.ai/@fivethirtyeight/antiquities-act

Reply:
I downloaded this a while ago from hCore from sugarscape but I don't now how this file got into there. I've never seen the Simulation you linked, sorry for this, that was by no mean on purpose!